spec/spec_helper.rb

```diff
@@ -34,7 +36,15 @@
     Rails.cache.clear
   end
 
-  config.after(:each, type: :feature) do |example|
+  config.before(:each, type: :system) do
+    driven_by :rack_test
+  end
+
+  config.before(:each, type: :system, js: true) do
+    driven_by :apparition
+  end
+
+  config.after(:each, type: :system) do |example|
```

Comment:
Shouldn't we conditionally load this only on older Solidus versions after https://github.com/solidusio/solidus/pull/3552 otherwise it's loaded twice when testing against Solidus master?

Reply:
Good for me 😄 
I added a new commit for this